### DISCUSSION TITLE
Welcome to fontations Discussions!

### DISCUSSION THREAD

**Discussion by anthrotype** [googlefonts/fontations#656] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ollimeier** [comment]
Excuse my for my spam in 'Issues' and thanks a lot for setting up 'Discussions'.

**Reply by anthrotype** [reply]
no, you're very welcome!
